pkg/profiler/cpu/maps.go

```diff
@@ -323,22 +323,24 @@ func (m *bpfMaps) setUnwindTable(pid int, ut unwind.UnwindTable) error {
 				return fmt.Errorf("write CFA register bytes: %w", err)
 			}
 
-			var CFARegister uint8
-			var RBPRegister uint8
-			var CFAOffset int16
-			var RBPOffset int16
+			var CfaRegister uint8
```

Comment:
Have you decided to change them back? 😅 I think the previous version is more correct in the conventional sense.

Reply:
haha no, I changed them because I remembered you mentioned that the current style wasn't the one we wanted for the project (https://github.com/parca-dev/parca-agent/pull/1004#discussion_r1017847289), but I probably forgot to change these back then.

Happy to rename them all to either `CFARegister` or `CfaRegister` style. Let me know which one we should use for the project 😄

Reply:
I don't mind, to be honest. It's hard to decide on the abbreviations. If most of the names are in this style, then let's stick with it.

Reply:
Same here! Leaving these with the updated style!